test/e2e/testing-manifests/cluster-dns/dns-frontend-pod.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dns-frontend
+  labels:
+    name: dns-frontend
+spec:
+  containers:
+    - name: dns-frontend
+      image: k8s.gcr.io/example-dns-frontend:v1
+      command:
+        - python
+        - client.py
+        - http://dns-backend.development.svc.cluster.local:8000
```

Comment:
We can't assume cluster.local is the domain in all clusters, I think this should use .svc as top domain

Reply:
Ah, it is replaced during the test
Scratch this comment

Reply:
thanks for the detailed review, anyway! :)